components/molecule/autosuggest/src/index.js

```diff
@@ -170,7 +170,6 @@ const MoleculeAutosuggest = ({
 
   const handleInputKeyDown = ev => {
     const {key} = ev
-    if (key !== 'ArrowDown') ev.stopPropagation()
```

Comment:
won't that be preventing from navigating the list of options?

Reply:
it still allows